ui/app/mainui/Popups.qml

```diff
@@ -1,28 +1,106 @@
-import QtQuick 2.14
+import QtQuick 2.15
 
 import AppLayouts.Chat.popups 1.0
+import AppLayouts.Profile.popups 1.0
+
 import shared.popups 1.0
+import shared.status 1.0
 
 import utils 1.0
 
 QtObject {
     id: root
 
-    /* required */ property var rootStore
+    required property var appMain
```

Comment:
This property name is imo a bit misleading bc it's just a parent of dynamically created components, not necessarily `appMain`. What about name e.g. `popupsParent`?

Reply:
Alright, yeah makes sense

Reply:
Done